example/src/main/java/io/netty/example/http/websocketx/server/WebSocketIndexPageHandler.java

```diff
@@ -56,7 +56,7 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpRequest req) thro
         }
 
         // Allow only GET methods.
-        if (!GET.equals(req.method())) {
+        if (!HttpMethod.GET.equals(req.method())) {
```

Comment:
I am confused... Isn't the existing code doing exactly this already ?

Reply:
> I am confused... Isn't the existing code doing exactly this already ?

I said this in my first comment :)

Reply:
sorry,i am wrong that i imported the GET method from other packages